spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultStreamService.java

```diff
@@ -217,6 +217,13 @@ private void updateStreamDefinitionFromReleaseManifest(String streamName, String
 				AuditOperationType.STREAM, AuditActionType.UPDATE, streamName, this.auditServiceUtils.convertStreamDefinitionToAuditData(streamDefinition));
 	}
 
+	@Override
+	public void scaleApplicationInstances(String streamName, String appName, String count, Map<String, String> properties) {
+		// Skipper expects app names / labels not deployment ids
+		logger.info(String.format("Scale %s:%s to %s with properties: %s", streamName, appName, count, properties));
+		// TODO: 1. Use Skipper's Scale API to send the scale request(s)
```

Comment:
@tzolov Do you think you can update this based on the Skipper PR?

Reply:
Just did further changes to a SkipperClient so it should be ready to use in that PR.